addons/website/static/src/js/editor/wysiwyg_multizone.js

```diff
@@ -574,6 +569,36 @@ var WysiwygMultizone = Wysiwyg.extend({
         this._focusedNode = node;
         this._super.apply(this, arguments);
     },
+    /**
+     * @override
+     */
+    _onShowDropdown: function (ev) {
+        $(ev.currentTarget).find('.dropdown-menu').addClass('show');
+        ev.preventDefault();
+        this.snippets.trigger('toggle_mega_menu_snippets', true);
+        $(ev.relatedTarget).dropdown('dispose');
+    },
+    /**
+     * Prevent dropdown closing when a contenteditable children is focused
+     *
+     * @override
+     */
+    _onHideDropdown: function (ev) {
+        if (ev.originalEvent &&
+                $(ev.target).has(ev.originalEvent.target).length &&
+                $(ev.originalEvent.target).is('[contenteditable]')) {
+            ev.preventDefault();
+        }
```

Comment:
Why is all this still there? Where is the `.dropdown('destroy')` we talked about? (to handle opening and closing our way) Did you try ?

Reply:
Of course. Did not work like you explained, so this is the simplified version I have.

Reply:
I implemented it when I explained it (it's like 10 lines of code) and it worked. Your solution is not easy to read but mainly: it still does not work in all cases (just tested and the menu still closes when I resize a column, just like I showed you last week).

Reply:
You should have shared it with me, just kidding :smile: Well, how am I suppose to do if everything is working with me ? I've just tried one more time now on runbot

Reply:
Well, retry. Tested on Firefox and Chrome, on your runbot and both fail. If it is really still working on your side, tell me I want to see it :)
BTW your runbot is red, maybe worth checking.

Reply:
I'll retry. Yes, runbot red is another thing I forgot to mention that I have to do. I quickly checked some days ago but did not find out yet.

Reply:
when you said .dropdown('destroy'), you meant .dropdown('dispose'), right ?

Reply:
Yes